### PR TITLE
Add sea surface albedo

### PR DESCRIPTION
**Pull request by tsaoyu**
pvlib python pull request guidelines
====================================

Thank you for your contribution to pvlib python!

You may submit a pull request with your code at any stage of completion, however, before the code can be merged the following items must be addressed:

 - [x] Closes issue #xxxx
 - [x] Fully tested. Added and/or modified tests to ensure correct behavior for all reasonable inputs. Tests must pass on the TravisCI and Appveyor testing services.
 - [x] Code quality and style is sufficient. Passes ``git diff upstream/master -u -- "*.py" | flake8 --diff`` and/or landscape.io linting service.
 - [x] New code is fully documented. Includes sphinx/numpydoc compliant docstrings and comments in the code where necessary.
 - [ ] Updates entries to `docs/sphinx/source/api.rst` for API changes.
 - [x] Adds description and name entries in the appropriate `docs/sphinx/source/whatsnew` file for all changes.

Please don't hesitate to ask for help if you're unsure of how to accomplish any of the above. You may delete all of these instructions except for the list above.

Brief description of the problem and proposed solution (if not already fully described in the issue linked to above):

This commit added sea surface albedo to ` SURFACE_ALBEDOS`. Technical reference come from [here](https://journals.ametsoc.org/doi/pdf/10.1175/1520-0469%281972%29029%3C0959%3AAOTSS%3E2.0.CO%3B2) 

Also a quick question on the permission of `irradiance.py` is`-rwxr-xr-x ` done for purpose?

### CODE REVIEW

**Comment by cwhanse** [comment]
@tsaoyu thanks for this addition.  Please update the signature of grounddiffuse to allow surface_type to have a new keyword value.

The value 0.04 appears to be a lower bound of some sort on sea surface albedo under clear skies.  I looked at the reference you linked, particularly at Table 5.  In my view, 0.06 is a more appropriate choice.  0.06 is also what is published at https://en.wikipedia.org/wiki/Albedo

We should also fix the references.  The albedo values are referenced in grounddiffuse, where the SURFACE_ALBEDO dict is used.  The reference comments should be moved to be adjacent to the dict's declaration.  Also, please change the reference to pvpmc.sandia.gov/...  to PVSyst v6 at http://files.pvsyst.com/help/albedo.htm, which is where we obtained the values published in the matlab code.  Please add the reference you linked as well.

Thanks!

**Comment by tsaoyu** [comment]
Sure, please review these commits.

Cheers

File: pvlib/irradiance.py [hunk 1]
```diff
@@ -489,9 +489,11 @@ def grounddiffuse(surface_tilt, ghi, albedo=.25, surface_type=None):
     The calculation is the last term of equations 3, 4, 7, 8, 10, 11, and 12.
 
     [2] albedos from:
```
Comment (cwhanse) [inline pvlib/irradiance.py]: Please move these comment lines to where SURFACE_ALBEDO is defined, unless that violates some python grammar rule (@wholmgren ??).  You can drop the '[2]'  Thanks again for these changes.

Comment (tsaoyu) [reply pvlib/irradiance.py]: Thank you both. I have added inline comments at line 20.

File: pvlib/irradiance.py [hunk 2]
```diff
@@ -30,7 +30,8 @@
                    'aluminum': 0.85,
                    'copper': 0.74,
                    'fresh steel': 0.35,
-                   'dirty steel': 0.08}
+                   'dirty steel': 0.08,
+                   'sea': 0.06}
```
[the remark this answers is not in the document]
Comment (cwhanse) [reply pvlib/irradiance.py]: That works too

**Comment by wholmgren** [comment]
Thanks @tsaoyu. I think we are ready to merge. I believe the Travis build failure on Python 3.6 is due to a new pandas 0.23 feature concerning preserving dict-order insertion.

**Comment by tsaoyu** [comment]
Thanks ! @wholmgren

**Pull request merged**